list.rb

```diff
@@ -0,0 +1,23 @@
+require_relative './MyEnumerable'
```

Comment:
- [ ] MyList class should implement #each method that yields all members of @list in succession. Kindly add it here
![image](https://user-images.githubusercontent.com/81932478/194727333-ceb736f2-c53e-4d67-86c6-871e6e407df8.png)